alg/marching_squares/square.h

```diff
@@ -269,16 +281,17 @@ struct Square
                 // The classical marching square algorithm says the ambiguity
                 // should be resolved between the two possible configurations by
                 // looking at the value of the center point. But in certain
-                // cases, this may lead to line contours from different levels
+                // cases, this may lead to line contours from different level, minLevels
```

Comment:
is this from a search replace or was it intended?
```suggestion
                // cases, this may lead to line contours from different level
```

Reply:
done